hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/ratis/TestOzoneManagerDoubleBufferWithOMResponse.java

```diff
@@ -390,7 +405,11 @@ private OMBucketCreateResponse createBucket(String volumeName,
     OmBucketInfo omBucketInfo =
         OmBucketInfo.newBuilder().setVolumeName(volumeName)
             .setBucketName(bucketName).setCreationTime(Time.now()).build();
-    return new OMBucketCreateResponse(omBucketInfo);
+    return new OMBucketCreateResponse(omBucketInfo, OMResponse.newBuilder()
```

Comment:
OMDummyCreateBucketResponse seems to be doing the same thing as OMBucketCreateResponse. Why do we need 2 different tests?

Reply:
This is added based on Arpit's comment in HDDS-1512. As we want to test OM Double Buffer Implementation without actual OM Responses.